dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -169,29 +169,30 @@ def readRecord(self,reader,writer,file_type,interfaces,row_number,job_id,fields,
             return {}, reduce_row, True, False, row_error_found
         return record, reduce_row, False, False, row_error_found
 
-    def writeToStaging(self, record, job_id, submission_id, passed_validations, interfaces, writer, row_number, file_type, error_list):
+    def writeToStaging(self, record, job_id, submission_id, passed_validations, writer, row_number, model, error_list):
         """ Write this record to the staging tables
 
         Args:
             record: Record to be written
             job_id: ID of current job
             submission_id: ID of current submission
             passed_validations: True if record has not failed first validations
-            interfaces: InterfaceHolder object
             writer: CsvWriter object
             row_number: Current row number
-            file_type: Type of file for current job
+            model: orm model for the current file
```

Comment:
Good call wrt passing in the most specific thing the function needs. We can always re-generalize later.